app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,37 @@
+<div class="w-[80%] flex flex-col items-center">
+
+<%= render @recipe %>
+<div class="flex gap-5">
+    <%= link_to "add ingredeints", new_user_recipe_recipe_food_path(params[:user_id],params[:id]), class:"border-slate-700 rounded-md border-2 p-2 bg-gray-50 text-slate-700 hover:bg-black hover:text-white text-center mb-4"%>
+    <%= link_to "general shooping list", user_recipe_shopinglists_path(params[:user_id],params[:id]), class:"border-slate-700 rounded-md border-2 p-2 bg-gray-50 text-slate-700 hover:bg-black hover:text-white text-center mb-4"%>
+</div>
+    <% @recipefoods.each do |foods| %>
+    sup
```

Comment:
- [ ] While you were correcting this section you accidentally added 👀 **sup** word, please remove it for a better user interface.